crates/bevy_ui/src/lib.rs

```diff
@@ -83,8 +83,7 @@ impl Plugin for UiPlugin {
             .add_system_to_stage(
                 CoreStage::PostUpdate,
                 widget::text_system
-                    .before(UiSystem::Flex)
-                    .after(ModifiesWindows),
+                    .after(UiSystem::Flex),
```

Comment:
@mockersf mentioned on discord this isn't a safe solution, I'm wondering if its worth making a new system for parent calculations and making it run after the UiSystem::Flex